README.md

```diff
@@ -9,21 +9,48 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/caos/zitadel)](https://goreportcard.com/report/github.com/caos/zitadel)
 [![codecov](https://codecov.io/gh/caos/zitadel/branch/master/graph/badge.svg)](https://codecov.io/gh/caos/zitadel)
 
-> This project is in alpha state. The API will continue breaking until version 1.0.0 is released
+> This project is in a alpha state. The application will continue breaking until version 1.0.0 is released
 
 ## What Is It
 
-It will be an IAM ;-)
+`ZITADEL` is a Cloud Native Identity and Access Management solution. All serverside componentes are written in `Go` and the management interface, called `console`, is written in `Angular`.
+
+We optimized ZITADEL for the usage as `service provider IAM`. By `service provider` we think of companies who build services for e.g SaaS cases. Often these companies would like to use a IAM where they can register there application and grant other people or companies the right to self manage a set of roles within that application.
 
 ## How Does It Work
 
+We built `ZITADEL` around the idea that the IAM should be easy to deploy and scale. That's why we tried to reduce external systems as much as possible.
+For example, `ZITADEL` is eventsourced but it does not rely on a pub/sub system to function. We instead built all the functionality right into one binary.
```

Comment:
Instead we built... (not: We instead built...)

Reply:
```suggestion
For example, `ZITADEL` is eventsourced but it does not rely on a pub/sub system to function. Instead we built all the functionality right into one binary.
```